packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -1385,6 +1584,7 @@ class StripeHelper {
       throw error;
     }
 
+    // @ts-ignore
```

Comment:
Stripe doesn't like key access like this, and without full TypeScript capabilities its not reasonable to express this in a type-safe manner, thus the ignore.